src/app/app.component.ts

```diff
@@ -44,7 +44,6 @@ import {
   BBAuthClientFactory
 } from '@skyux/auth-client-factory';
 
-require('style-loader!@skyux/theme/css/sky.css');
```

Comment:
Removing this and putting it in the module generator (so that it's handled like all the other `app.styles` style sheets in `skyuxconfig.json`).